### DISCUSSION TITLE
Why were PBR & GLTF removed?

### DISCUSSION THREAD

**Discussion by Nolram12345** [raysan5/raylib#3271] [Q&A]
Hello! 
I was looking through raylib features, and found "Flexible Materials system, supporting classic maps and **PBR maps**" listed. I was curious about this so I went to check, however I could not find any reference to something like that in raylib. So I then went searching for it, and found out that raylib used to have a PBR example and a GLTF importer, but they were removed in raylib 4. 

Why was this done? Would it be possible to see these features again in the future?

**Comment by Nolram12345** [comment]
I see, thank you very much for the clarification!
GLTF does seem to be a thing, but it appears the actual shaders for rendering GLTF models with PBR are not present (anymore), and there is no example of how to implement such either.

**Comment by raysan5** [comment]
@Nolram12345 raylib `Model`/`Mesh`/`Material` are prepared to support PBR maps but the shaders are not provided, neither an examples. There was an example in the past but it was broken and nobody ready to maintain it so finally it was removed.

Feel free to provide a PBR example, I'm sure many users will be very happy with it.

**Comment by Nolram12345** [comment]
Well, my exact issue was that I was looking for what a PBR implementation with raylib would look like. I can't provide an example because I am not aware how to implement something like this in this context, because I'm not a raylib expert and neither is a sample for it provided. That's why I was asking for one in the first place.
